qc/qc.xml

```diff
@@ -10,111 +10,108 @@
             plink
         </requirement>
     </requirements>
-    <command>
-        <![CDATA[
-
-
-#if '$performLDcond.performLD':
-    bash '$__tool_directory__/qc.sh' '$vcf_file' '$pheno_file' '$maf_threshold' '$hwe_threshold' '$mgcf_threshold' '$max_heterozygosity' '$performLDcond.window_size' '$performLDcond.step_size' '$performLDcond.ld_threshold' '$pbed_output' '$__tool_directory__' '$performLDcond.performLD' '$fid';
-#else
-    bash '$__tool_directory__/qc.sh' '$vcf_file' '$pheno_file' '$maf_threshold' '$hwe_threshold' '$mgcf_threshold' '$max_heterozygosity' '$pbed_output' '$__tool_directory__' '$performLDcond.performLD' '$fid';
-#end if
-
-
+    <command><![CDATA[
+    bash '$__tool_directory__/qc.sh' 
+        '$vcf_file'
+        '$pheno_file'
+        '$maf_threshold'
+        '$hwe_threshold'
+        '$mgcf_threshold'
+        '$max_heterozygosity'
+        #if '$performLDcond.performLD':
+            '$performLDcond.window_size'
+            '$performLDcond.step_size'
+            '$performLDcond.ld_threshold'
+        #end if
+        '$pbed_output'
+        '$__tool_directory__'
+        '$performLDcond.performLD'
+        '$fid'
 ]]>
     </command>
     <inputs>
-
-        <param name="vcf_file" type="data" format="vcf_bgzip" />
-        <param name="fid" label="Family ID column" type="boolean" checked="true" help="Yes if phenotype file has Family ID column, else No. (The format is what is expected by Plink1.9)"/>
-        <param name="pheno_file" type="data" format="tabular" />
-        <param name="maf_threshold" type="float" value="0" label="maf Threshold value" help="Set the threshold for minor allele frequency" />
-        <param name="hwe_threshold" type="float" value="0" label="hwe Threshold value" help="Set the threshold for Hardy-Weinberg equilibrium" />
-        <param name="mgcf_threshold" type="float" value="0" label="mgcf Threshold value" help="Set the threshold for minor genotypic class frequency" />
-        <param name="max_heterozygosity" type="float" value="1" label="max heterozygosity Threshold value" help="Set the threshold for heterozygosity" />
+        <param name="vcf_file" type="data" format="vcf_bgzip"/>
+        <param name="fid" type="boolean" checked="true" label="Family ID column" help="Yes if phenotype file has Family ID column, else No. (The format is what is expected by Plink1.9)"/>
+        <param name="pheno_file" type="data" format="tabular"/>
+        <param name="maf_threshold" type="float" value="0" label="maf Threshold value" help="Set the threshold for minor allele frequency"/>
```

Comment:
can you add here min/max values? and to other integer and float parameters if possible